commit d4fdfe3402c6f91f9fb184d19c05c29deae98266 (refs/changes/27/2927/1)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2017-11-08T16:39:58-08:00 (1 year, 11 months ago)
    
    MANATEE-387 JPC manatee pg_dump.sh duration impedes month-end billing cycle

diff --git a/pg_dump/pg_backup_common.sh b/pg_dump/pg_backup_common.sh
index c329a00..f02e651 100644
--- a/pg_dump/pg_backup_common.sh
+++ b/pg_dump/pg_backup_common.sh
@@ -161,6 +161,7 @@ function backup ()
             sudo -u postgres pg_dump -p 23456 moray -a -t $i | gsed 's/\\\\/\\/g' | sqlToJson.js | gzip -1 > $dump_file
             [[ $? -eq 0 ]] || fatal "Unable to dump table $i"
         done
+        move_pg_dumps
         #
         # If we have napi_ips_* tables, dump them to separate files based on the
         # first two hex characters of their uuid, since there will be many
@@ -182,6 +183,7 @@ function backup ()
                 fi
             done
         fi
+        move_pg_dumps
         rm $schema
         [[ $? -eq 0 ]] || fatal "unable to remove schema"
     fi
diff --git a/pg_dump/pg_dump.sh b/pg_dump/pg_dump.sh
index 0f063ac..5422212 100755
--- a/pg_dump/pg_dump.sh
+++ b/pg_dump/pg_dump.sh
@@ -53,7 +53,7 @@ if [[ $? = '1' ]]; then
     check_lock
     mount_data_set
     backup 'JSON'
-    move_pg_dumps
+    # MANATEE-387: backup() moves the logs into place for upload.
 else
     echo "not performing backup, not lowest peer in shard"
     exit 0
